lib/cmd/fh3/appforms/forms/import.js

```diff
@@ -0,0 +1,18 @@
+var url = '/api/v2/appforms/forms/import';
+var upload = require('../../../../utils/request').formsImportZipFileUpload;
+
+module.exports = {
+  'desc': 'Imports a zip file containing definitions for multiple forms, intended to be used with *export*',
+  'examples': [{cmd: 'fhc appforms forms import --file=<path to zip file>', desc: 'Imports forms contained in the zip file'}],
+  'demand': ['file'],
+  'alias': {},
+  'describe': {
+    'file': 'Path to a file'
+  },
+  'url': url,
+  'method': 'post',
```

Comment:
Yeah, the POST is handled by `request`, I wasn't sure if leaving it on the command metadata was relevant for documentation or other tooling that runs on top of the fh3 command structure.